src/wymeditor/editor/keyboard.js

```diff
@@ -0,0 +1,70 @@
+"use strict";
+WYMeditor.Keyboard = function (wym) {
+    var keyboard = this;
+
+    keyboard._wym = wym;
+
+    keyboard.combokeys = new WYMeditor.EXTERNAL_MODULES.Combokeys(wym._doc);
+
+    return keyboard;
+};
+
+WYMeditor.Keyboard.DEFAULT_KEYBOARD_SHORTCUTS = [
+    {
+        combo: "mod+b",
+        cb: function () {
+            this.exec(WYMeditor.EXEC_COMMANDS.BOLD);
+            // Prevents native action. Caution: not covered by tests.
+            return false;
+        }
+    },
+    {
+        combo: "mod+i",
+        cb: function () {
+            this.exec(WYMeditor.EXEC_COMMANDS.ITALIC);
+            // Prevents native action. Caution: not covered by tests.
+            return false;
+        }
+    },
+    {
+        combo: "mod+k",
```

Comment:
Right. De–facto standard is via tooltips on buttons. Aded complexity is displaying the correct ones according to the OS.